flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorITCase.java

```diff
@@ -531,6 +531,34 @@ public void testLocationWithComma() throws Exception {
         }
     }
 
+    @Test
+    public void testReadHiveDataWithEmptyMapForHiveShim20X() throws Exception {
```

Comment:
We can only run this test for hive <=2.0.0. Something like `Assume.assumeTrue(HiveShimLoader.getHiveVersion().compareTo("2.0.0") <= 0)`